torch/_export/serde/schema.py

```diff
@@ -27,6 +27,10 @@ class ScalarType(IntEnum):
     COMPLEXDOUBLE = 11
     BOOL = 12
     BFLOAT16 = 13
+    FLOAT8_E5M2 = 23
+    FLOAT8_E4M3FN = 24
+    FLOAT8_E5M2FNUZ = 25
+    FLOAT8_E4M3FNUZ = 26
```

Comment:
Why there is a gap in numbering?

Reply:
It's because the numbers need to match what's [here](https://github.com/pytorch/pytorch/blob/cab90b0049f75a1c6d9ca034e5011fa170a944e8/c10/core/ScalarType.h#L80-L83)! Thanks for helping @malfet, I was stuck